lib/utils/constants.dart

```diff
@@ -0,0 +1,6 @@
+class Constants {
+  static const int status = 200;
```

Comment:
Instead to add a this constants, you can importa HttpStatus and then you can find all the status code that you need